Cubical/Categories/Category.agda

```diff
@@ -1,17 +1,17 @@
 {-
   Definition of various kinds of categories.
 
-  This library follows the UniMath terminology, that is:
+  This library partially follows the UniMath terminology:
 
   Concept              Ob C   Hom C  Univalence
 
-  Precategory          Type   Type   No
+  Wild category        Type   Type   No
   Category             Type   Set    No
   Univalent Category   Type   Set    Yes
 
   The most useful notion is Category and the library is hence based on
   them. If one needs precategories then they can be found in
-  Cubical.Categories.Category.Precategory
+  Cubical.WildCat (so it's not considered part of the Categories sublibrary!)
```

Comment:
Good catch here. There is a mention of precategories on line 13 that should also be changed to "wild categories"